src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -200,4 +203,75 @@ module.exports = class LevelPlane extends Array {
 
     return this[myIndex].blockType;
   }
+
+  getRedstone() {
+    this.redstoneList = [];
+    this.redstoneListON = [];
+    for (let i = 0; i < this.length; ++i) {
+      if (this[i].blockType.substring(0,12) === "redstoneWire") {
+        this[i].isPowered = false;
+        this.redstoneList.push(i);
+      }
+    }
+    for (let i = 0; i < this.length; ++i) {
+      if (this[i].blockType === "railsRedstoneTorch") {
+        let y = Math.floor(i / this.height);
+        let x = i - (y * this.height);
+        let pos = [x,y];
+        this.redstonePropagation(pos);
+      }
+    }
+
+    for (let obj in this.redstoneList) {
+      console.log("The block at index: " + this.levelModel.controller.levelView.indexToCoordinates(this.redstoneList[obj]) + " is a " + this[this.redstoneList[obj]].blockType + ". This block is NOT charged.");
```

Comment:
I wonder if it's better to store a list of positions, so we don't have to convert from index to position?

Ideally we'd use `position` everywhere possible, only converting to index when looking up values in the internal array.  No external consumers should need to know the implementation of the underlying data structure.